en-US/firefox/all-unified.lang

```diff
@@ -0,0 +1,143 @@
+## NOTE: https://www-dev.allizom.org/firefox/all/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/all/
+
+
+# HTML page title. Replace "English (US)" with your local language.
+;Download the Firefox Browser in English (US) and more than 90 other languages
+Download the Firefox Browser in English (US) and more than 90 other languages
+
+
+# HTML page description, also used as the introductory text.
+;Everyone deserves access to the internet — your language should never be a barrier. That’s why — with the help of dedicated volunteers around the world — we make the Firefox Browser available in more than 90 languages.
+Everyone deserves access to the internet — your language should never be a barrier. That’s why — with the help of dedicated volunteers around the world — we make the Firefox Browser available in more than 90 languages.
+
+
+;Choose which Firefox Browser to download in your language
+Choose which Firefox Browser to download in your language
+
+
+;Check the system requirements
+Check the system requirements
+
+
+;Release notes
+Release notes
+
+
+;Source code
+Source code
+
+
+;Firefox Privacy Notice
+Firefox Privacy Notice
+
+
+;Need help?
+Need help?
+
+
+;Which browser would you like to download?
+Which browser would you like to download?
+
+
+;Learn about Firefox browsers
+Learn about Firefox browsers
+
+
+# Used as an accessible label for a help button. The text is replaced with a "?" icon.
+;Get help
+Get help
+
+
+;You are about to download:
+You are about to download:
+
+
+;Browser:
+Browser:
+
+
+;Platform:
+Platform:
+
+
+;Language:
+Language:
+
+
+;Sorry, we couldn’t find the download you're looking for. Please try again, or select a download from the list below.
+Sorry, we couldn’t find the download you're looking for. Please try again, or select a download from the list below.
```

Comment:
nit: `couldn’t` uses a smart quote, but `you're` looks like a straight quote.

Reply:
Good catch, I'll fix this in the template.